public/app/features/plugins/extensions/getExtensionValidationResults.ts

```diff
@@ -0,0 +1,61 @@
+import { PluginContextType } from '@grafana/data';
+
+import * as errors from './errors';
+import { ExtensionsLog, log } from './logs/log';
+import { isGrafanaDevMode } from './utils';
+import { isExtensionPointIdValid, isExtensionPointMetaInfoMissing } from './validators';
+
+interface GetExtensionValidationResultsOptions {
+  extensionPointId: string;
+  isLoadingAppPlugins: boolean;
+  pluginContext: PluginContextType | null;
+}
+
+interface ExtensionsValidationResult {
+  isLoading: boolean;
+  results: [];
+}
+
+type GetExtensionValidationResultsResult = {
+  result: ExtensionsValidationResult | null;
+  pointLog: ExtensionsLog;
+};
+
+export function getExtensionValidationResults({
```

Comment:
Nit: I would probably name it slightly differently, as strictly speaking this is an _extension point_ (content user) and not an _extension_ (provided content), e.g. `getExtensionPointValidationResults(` or `validateExtensionPoint(`.